src/spikeinterface/preprocessing/motion.py

```diff
@@ -511,6 +513,19 @@ def load_motion_info(folder):
         else:
             motion_info[name] = None
 
-    motion_info["motion"] = Motion.load(folder / "motion")
-
+    if (folder / "motion").is_dir():
+        motion = Motion.load(folder / "motion")
+    else:
+        warnings.warn("Trying to load Motion from the legacy format")
+        required_files = ["spatial_bins.npy", "temporal_bins.npy", "motion.npy"]
+        for required_file in required_files:
+            if not (folder / required_file).is_file():
+                raise IOError("The provided folder is not a valid motion folder")
+        spatial_bins_um = np.load(folder / "spatial_bins.npy")
+        temporal_bins_s = [np.load(folder / "temporal_bins.npy")]
+        displacement = [np.load(folder / "motion.npy")]
+
+        motion = Motion(displacement=displacement, temporal_bins_s=temporal_bins_s, spatial_bins_um=spatial_bins_um)
```

Comment:
```suggestion
        motion = Motion(displacement=displacement, temporal_bins_s=[temporal_bins_s], spatial_bins_um=[spatial_bins_um])
```

Reply:
No :) I'm ok making the list later if it's better readable

Reply:
Actually, let's leave it as is!